src/rpc/mining.cpp

```diff
@@ -22,8 +22,11 @@
 #include <rpc/mining.h>
 #include <rpc/server.h>
 #include <txmempool.h>
-#include <util.h>
-#include <utilstrencodings.h>
+#include <util/fees.h>
+#include <util/strencodings.h>
+#include <util/system.h>
+#include <util/validation.h>
+#include <validation.h>
```

Comment:
```
Duplicate include(s) in src/rpc/mining.cpp:
#include <validation.h>
```
https://gitlab.com/dashpay/dash/-/jobs/1375020079#L55
should drop this line or (even better) a similar include at line 15